federatedscope/core/fed_runner.py

```diff
@@ -427,3 +432,62 @@ def _handle_msg(self, msg, rcv=-1):
                 self.client[each_receiver].msg_handlers[msg.msg_type](msg)
                 self.client[each_receiver]._monitor.track_download_bytes(
                     download_bytes)
+
+    def check(self):
+        """
+        Check the completeness of Server and Client.
+
+        Returns:
+
+        """
+        if self.cfg.check_completeness:
```

Comment:
I don't think it is necessary to indent such a huge block of code

Reply:
Update accordingly.